sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuOrcFileFormat.scala

```diff
@@ -227,6 +229,9 @@ class GpuOrcWriter(
     val builder = SchemaUtils
       .writerOptionsFromSchema(ORCWriterOptions.builder(), dataSchema, nullable = false)
       .withCompressionType(CompressionType.valueOf(OrcConf.COMPRESS.getString(conf)))
+    orcStripeSizeRows.foreach { orcStripeSizeRows =>
```

Comment:
NIT: Give a different name for the defined stripe size
```suggestion
    orcStripeSizeRows.foreach { ss =>
```